manager/state/raft/raft.go

```diff
@@ -1703,14 +1707,23 @@ func (n *Node) processInternalRaftRequest(ctx context.Context, r *api.InternalRa
 	select {
 	case x, ok := <-ch:
 		if !ok {
+			// Wait notification channel was closed. This should only happen if the wait was cancelled.
+			log.G(ctx).Errorf("wait cancelled, likely because node %x lost leader position. Wait channel closed with nothing to read.", n.opts.ID)
+			if atomic.LoadUint32(&n.signalledLeadership) == 1 {
+				log.G(ctx).Errorf("Wait cancelled but node %x is still a leader.", n.opts.ID)
```

Comment:
I don't mean to pontificate on this too much, but can we make this message textually different from the one below, instead of just using lower and upper case as the difference? By convention, all messages are lower case.

cc @aaronlehmann if you have ideas.

Reply:
The sadness of not including file names/line numbers in log messages :(

Reply:
BTW, if your concern is to differentiate the two cases, it is still possible because there's another log before this one. @nishanttotla